<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="icon" href="icon_path" type="image/icon type">
    <link rel="icon" href="ok image/virus.png" type="image/x-icon" />
  
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

    <style>


    
    /* Responsive design for smaller screens */
    @media (max-width: 768px) {
        .footer-container {
            flex-direction: column;
            text-align: center;
        }
    
        .footer-links ul {
            margin-bottom: 15px;
        }
    
        .footer-social ul {
            margin-bottom: 15px;
        }
    }
    @media (max-width: 768px) {
        .sss {
            flex-direction: column;
            text-align: center;
        }
        .ss{
          flex-direction: column;
            text-align: center;
        }
      }
      .sh{
        margin: 50px;
      }


    </style>
</head>
<body>
 
  <nav class="navbar navbar-expand-lg shadow rounded" style="background-color: #EFEFEF;">
    <div class="container-fluid">
      <a class="navbar-brand" href="#"><img src="ok image/virus.png" height="60px" width="60px" alt=""></a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse justify-content-center" id="navbarNav">
        <ul class="navbar-nav">
          <li class="nav-item">
            <a class="nav-link text-secondary fw-bold fs-6" aria-current="page" href="index.html">Home</a>
          </li>
          <li class="nav-item">
            <a class="nav-link ps-5 text-secondary fw-bold fs-6" href="about-us.html">About Us</a>
          </li>
          <li class="nav-item">
            <a class="nav-link ps-5 text-secondary fw-bold fs-6" href="types.of .virus.html">Types of Viruses</a>
          </li>
          <li class="nav-item">
            <a class="nav-link ps-5 text-secondary fw-bold fs-6" href="year-wise-listing.html">Year Wise Listing</a>
          </li>
          <li class="nav-item">
            <a class="nav-link ps-5 text-secondary fw-bold fs-6" href="contact.html">Contact Us</a>
          </li>
        </ul>
      </div>
    </div>
  </nav>  
       <div class="card text-bg-dark">
        <img src="ok image/Mimivirus.png"height="500px" class="card-img" alt="...">
        <div class="card-img-overlay">
          <!-- <h1 class="card-title">VIRUS INFO:</h1>
          <p class="card-text">A virus is small infection agent that replicates only inside the living cells of an organism. <br> viruses can affect all types of life forms. from animals and plants to microorganisms including <br> bacteria and archaea .</p>
          -->
        </div>
      </div>
      
      
      <div class="sh">
      <div class="f2">
        <div class="f1"><h2><b><i>Mimi virus :</b></i></h2></div><br>
        <div class="card mb-6" style="max-width: 100%; border-style: none;">
         <div class="row g-0">
           <div class="col-md-6">
             <h2 class="card-title">  Discovery of the Giant Mimivirus :</h2>
             <p class="card-text" ><h5>First observed in 1992, Acanthamoeba polyphaga mimivirus (APMV) presents an interesting story of scientific inquiry. During the investigation of a pneumonia outbreak, researchers noticed particles resembling Gram-positive bacteria residing within amoebae isolated from a water-cooling tower. Because other bacterial species that cause pneumonia, like Legionella pneumophila, reside within amoebae (Rowbotham 1980), the investigators hypothesized that they had found another pneumonia-causing bacterium. Subsequent research, however, showed that they had identified not a bacterium, but a giant virus (La Scola et al. 2003).</h5></p>
            
           </div>
           <div class="col-md-6">
             <div class="card-body">
                 <img src="ok image/fusion-medical-animation-npjP0dCtoxo-unsplash.jpg" class="img-fluid rounded-start " alt="...">
             </div>
           </div>
         </div>
       </div>
         <h5>Mimivirus is a DNA virus that infects amoebae. It is an extremely large (400 nm) particle that stains Gram-positive and has a genome size of 1.2 Mb [1]. The genome contains genes not previously reported in viruses, encoding proteins involved in translation of proteins and components of metabolic pathways. <br>
            <!-- <h2><b> <i>  </b></h2>  -->
             By: David R. Wessner, Ph.D. (Dept. of Biology, Davidson College) © 2010 Nature Education 
             Citation: Wessner, D. R. (2010) Discovery of the Giant Mimivirus. <br>  Nature Education 3(9):61
             Share Using RedditStumbleUponShare with Google+Share with TwitterShare with FacebookEmailPrintBookmark
             Mimivirus is the largest and most complex virus known.  <br> Is it an evolutionary bridge between nonliving viruses and living organisms, or is it just an anomaly? <br>
             Aa Aa Aa
             Viruses are small and fairly simple. <br> At least that's what many people probably assume. With the discovery of Mimivirus — the largest, most complex virus currently known — these assumptions may need to be reevaluated. <br>  This giant virus has a much larger size and bigger genome than any other known virus. Analysis of this intriguing virus may shed light on basic questions of viral evolution and, perhaps, the origins of life. <br>
             
             When most people think about viruses, they probably first think about diseases. <br>  Flu, polio, AIDS. We all know that viruses have caused immeasurable suffering throughout human history. <br>  And when asked to describe a virus, most people probably would say, "small." Indeed, viruses were first identified as filterable infectious agents (Beijerinck 1898; Ivanofsky 1892) — entities that could cause an infection and pass through a filter small enough to exclude almost all bacteria.  <br> 
             
             Their small size and inability to carry out translation — the formation of polypeptides from messenger RNA — often are highlighted as two defining characteristics of viruses. They seem to share little else. Some viruses have RNA genomes, and some have DNA genomes. Some have single-stranded genomes, and others have double-stranded genomes. Some possess envelopes, while others do not. And structurally, viruses display a wonderful array of shapes. The discovery of the giant Mimivirus certainly forces us to rethink our assumption that viruses are small. Analysis of this virus also may cause us to rethink our assumptions about the tree of life. <br>
             
             Discovery and Analysis of the Giant Mimivirus
             
             An electron micrograph shows an amoeba that has been invaded by a Mimivirus. <br> A scale bar represents 2 microns, and the amoeba measures approximately 10 microns across. A circular, approximately 2-micron region in the amoeba's center is the nucleus. A darkly-shaded circular region of similar size is the viral factory. Congregating alongside the viral factory's border are approximately 20 Mimivirus virions. These are smaller, 0.5 to 1 micron circles, each with a circular black patch in its center. <br> 2-4 virions are separated from the region surrounding the viral factory and scattered around the cell. <br>
             View Full-Size ImageFigure 1 <br>
             First observed in 1992, Acanthamoeba polyphaga mimivirus (APMV) presents an interesting story of scientific inquiry. <br> During the investigation of a pneumonia outbreak, researchers noticed particles resembling Gram-positive bacteria residing within amoebae isolated from a water-cooling tower. <br> Because other bacterial species that cause pneumonia, like Legionella pneumophila, reside within amoebae (Rowbotham 1980), the investigators hypothesized that they had found another pneumonia-causing bacterium. Subsequent research, however, showed that they had identified not a bacterium, but a giant virus (La Scola et al. 2003). <br>
             
             Indeed, the most dramatic feature of this virus is its size. Researchers, in fact, named it Mimivirus — short for "mimicking microbe" — to reflect its large size and apparent Gram-staining properties (Figure 1). The virus has a capsid diameter of 400–500 nanometers (nm) and a total particle diameter, including fibers extending out from the capsid, of approximately 750 nm (Xiao et al. 2005). <br> A second strain of APMV, currently referred to as "mamavirus," may be even larger (La Scola et al. 2008). <br> These APMV strains dwarf all other known viruses and exceed the size of several well-characterized bacteria. <br> Poliovirus particles, in comparison, have a diameter of only 30 nm, which is roughly 10,000 times smaller than a grain of salt. Even the relatively large, brick-shaped poxviruses, such as smallpox virus, usually measure about 200 nm wide and 300 nm long — not even half the size of Mimivirus. <br> At approximately 1.2 million base pairs (Raoult et al. 2004), the linear, double-stranded DNA genome of Mimivirus far exceeds the size of any other known virus and a number of bacteria. Again, as a point of comparison, poliovirus has a genome of only 7,500 nucleotides, and the smallpox virus genome is about 200,000 nucleotides long. <br>
             <br><br>
            <b> The Mimivirus Genome</b><br><br>
             While the size of Mimivirus certainly stretches our common perceptions of viruses, researchers have been more interested in the types of genes found within the Mimivirus genome. This information, they reasoned, could help us understand the evolutionary history of Mimivirus and, perhaps, shed some light on the more basic question of the history of all viruses. To carry out this type of analysis, genes within the Mimivirus genome would need to be compared to genes of other organisms. When organisms share similar genes, we can hypothesize that they share an evolutionary heritage.
             
             Using standard sequencing techniques (Adams 2008), researchers first sequenced the entire 1,181,404-base-pair Mimivirus genome (Raoult et al. 2004) and then analyzed the sequence. <br> They predicted that the genome contains 911 potential protein-coding genes, or open reading frames (ORFs; Raoult et al. 2004). <br> The human influenza virus genome, by contrast, encodes only 11 proteins; and the smallpox genome encodes fewer than 200 proteins. <br>
             
            <br> To further analyze this large virus, investigators next compared the putative Mimivirus genes to known gene sequences from other viruses and cellular organisms. <br> The results surprised them. Mimivirus appears to contain many ORFs previously identified in members of the nucleocytoplasmic large DNA virus (NCLDV) group, a group that also includes human pathogens like herpesviruses and smallpox virus. <br> However, the exact constellation of these genes in the Mimivirus genome does not match any of the known NCLDVs. <br> This result led investigators to postulate that Mimivirus probably represents the first identified member of a new class of NCLDVs (Raoult et al. 2004). <br>
             
             Although the presence of these NCLDV genes in Mimivirus is interesting, it might not seem surprising. <br> Given the structural similarities between Mimivirus and other large DNA viruses, we should expect these viruses to contain some similar genes. The identification of other genes, however, clearly surprised the microbiology community. <br> As the authors of the study note, the Mimivirus genome includes "many genes never before identified in a viral genome" (Raoult et al. 2004). More specifically, the Mimivirus genome appears to contain many genes needed for processes thought to be hallmarks of life: protein translation and metabolism. <br>
             
           <br>  Specifically, the Mimivirus genome contains a number of ORFs showing strong homology to genes that encode aminoacyl-tRNA synthetases, the enzymes that link amino acids to their appropriate tRNA molecules. <br> Mimivirus also possesses several genes showing homology to translation initiation factors, other key components of translation. <br> In addition to these translation-associated genes, the Mimivirus genome includes genes associated with metabolic pathways, DNA repair, and protein folding. <br> Researchers now are investigating whether the virus utilizes proteins produced from these putative genes. <br> These investigations may reveal whether the genes are essential for Mimivirus replication.
             
             Certain other NCLDVs have been shown to possess a few translation-related genes. Among viruses, though, many of these genes appear to be unique to Mimivirus, and no other identified virus contains anywhere near the same number of these genes. If Mimivirus relies on its host for translation, then why would it possess so many genes related to this process? We still do not have a clear answer to this fundamental question. <br>
             
             Evolutionary History of Mimivirus <br>
             
             
             View Full-Size ImageFigure 2
             What can we conclude from this information? Like all other viruses, Mimivirus lacks ribosomes and depends on its host for translation. Mimivirus, however, contains a much more complete repertoire of translation-associated genes than does any other known virus. In interpreting this information, the authors of the initial report characterizing the Mimivirus genome postulated that Mimivirus may have evolved from a more independent ancestor (Raoult et al. 2004). Over time, they argue, Mimivirus lost some genes associated with translation as it became more dependent on its host. <br> The translation-like genes that we see in its genome today are relics of a previously intact translation machinery.  <br>Mimivirus, in other words, may represent evidence in support of the regressive model of virus origins. <br> Or perhaps, they note, Mimivirus represents a new branch on the tree of life, distinct from Archaea, Bacteria, and Eukarya. <br> Raoult and Forterre (2008) even argue that we should reclassify all biological entities into two major groups: the ribosome-encoding organisms (archaea, bacteria, and eukaryotes), or REOs; and the capsid-encoding organisms (viruses), or CEOs (Figure 2).
             
             Of course, the biological world rarely presents us with clear-cut answers to our questions. <br> In a separate analysis of the Mimivirus genome, other researchers provide a different explanation for the evolutionary history of this unusual virus. Moreira and Brochier-Armanet (2008) argue that the Mimivirus genes related to translation, DNA repair, and metabolism do not represent the remains of previously intact processes. Rather, they argue, Mimivirus acquired these genes over time via horizontal gene transfer (HGT). These genes, in other words, were acquired by Mimivirus from other organisms (Figure 3). <br> To support this assertion, these researchers note that some ORFs in the Mimivirus genome seem to show strongest homology to ORFs in bacteria, while other Mimivirus ORFs show strongest homology to ORFs in eukaryotes. <br> To logically explain this observation, they argue that some genes in Mimivirus may have been acquired from the amoeba host of Mimivirus. <br> Other Mimivirus genes may have been acquired from parasitic bacteria residing within the amoeba.<br> The amoeba, then, may serve as a genetic mixing bowl from which Mimivirus has gathered a complex set of genes (Moreira & Brochier-Armanet 2008). <br>
             
              <br>As often happens in scientific inquiries, the investigation of one question leads to unexpected findings and a host of new questions. As noted at the beginning of this article, the discovery of Mimivirus began during the investigation of a pneumonia outbreak. We still do not know, however, whether <b> Mimivirus causes pneumonia </b>. In one study, La Scola and colleagues (2005) reported that people with community-acquired pneumonia were more likely to have antibodies to Mimivirus than were people who did not have the disease. Although one can interpret this result as indicating that people with pneumonia are more likely to have been infected with Mimivirus, this result alone does not indicate that Mimivirus causes pneumonia. In another study (Khan et al. 2007), researchers showed that mice experimentally inoculated with Mimivirus developed tissue damage consistent with the development of pneumonia. <br> Again, we cannot conclude from this result that Mimivirus causes pneumonia. The result merely indicates that the virus, under certain experimental conditions, can cause tissue damage in mammals.
             
             Though the discovery and subsequent characterization of Mimivirus have not directly answered the original question about pneumonia, the studies of Mimivirus have opened the door to a whole host of other questions about the evolutionary history of viruses and the origins of life. Future studies of APMV strains certainly will provide us with important insight into these basic topics. <br> </h5> <br>
     

              </div>
             <footer class="text-center bg-body-tertiary">
              <!-- Grid container -->
              <div class="container pt-4">
                <!-- Section: Social media -->
                <section class="mb-4">
                  <!-- Facebook -->
                  <a
                    data-mdb-ripple-init
                    class="btn btn-link btn-floating btn-lg text-body m-1"
                    href="#!"
                    role="button"
                    data-mdb-ripple-color="dark"
                    ><i class="fa fa-facebook"></i
                  ></a>
            
                  <!-- Twitter -->
                  <a
                    data-mdb-ripple-init
                    class="btn btn-link btn-floating btn-lg text-body m-1"
                    href="#!"
                    role="button"
                    data-mdb-ripple-color="dark"
                    ><i class="fa fa-twitter"></i
                  ></a>
            
                  <!-- Google -->
                  <a
                    data-mdb-ripple-init
                    class="btn btn-link btn-floating btn-lg text-body m-1"
                    href="#!"
                    role="button"
                    data-mdb-ripple-color="dark"
                    ><i class="fa fa-google"></i
                  ></a>
            
                  <!-- Instagram -->
                  <a
                    data-mdb-ripple-init
                    class="btn btn-link btn-floating btn-lg text-body m-1"
                    href="#!"
                    role="button"
                    data-mdb-ripple-color="dark"
                    ><i class="fa fa-instagram"></i
                  ></a>
            
                  <!-- Linkedin -->
                  <a
                    data-mdb-ripple-init
                    class="btn btn-link btn-floating btn-lg text-body m-1"
                    href="#!"
                    role="button"
                    data-mdb-ripple-color="dark"
                    ><i class="fa fa-linkedin"></i
                  ></a>
                  <!-- Github -->
                  <a
                    data-mdb-ripple-init
                    class="btn btn-link btn-floating btn-lg text-body m-1"
                    href="#!"
                    role="button"
                    data-mdb-ripple-color="dark"
                    ><i class="fa fa-github"></i
                  ></a>
                </section>
                <!-- Section: Social media -->
              </div>
              <!-- Grid container -->
            
              <!-- Copyright -->
              <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.05);">
                © 2024 Copyright:
                <a class="text-body" href="index.html">Viruses</a>
              </div>
              <!-- Copyright -->
            </footer>
            
          
          
          <script>
              // Initialization for ES Users
          import { Ripple, initMDB } from "mdb-ui-kit";
          
          initMDB({ Ripple });
          </script>





</body>
</html>